misc/package/Data Files/MWSE/core/mwse/traveled/main.lua

```diff
@@ -17,10 +22,7 @@ local function travelStart(e)
 		-- first opening of the Travel Menu. Once again after a destination has been
 		-- selected, but before the player was teleported. This event is triggered
 		-- after the player is teleported to the destination.
-		timer.frame.delayOneFrame(function()
-			payload.cell = tes3.mobilePlayer.cell
-			event.trigger(tes3.event.traveled, payload)
-		end)
+		event.register(tes3.event.simulate, travelEnd, { doOnce = true })
```

Comment:
Great! As G7 mentioned you need to make sure that this event registration is discarded in the loaded event via event.isRegistered/unregister. I think its pretty much impossible for anyone to open a menu during that, but better be safe than sorry.

Reply:
Added the event.isRegistered/unregister part